docs/core/install/windows.md

```diff
@@ -220,6 +220,8 @@ Suppresses any attempts to restart.
 dotnet-sdk-9.0.100-win-x64.exe /install /quiet /norestart
 ```
 
+You can replace the `/install` option with `/uninstall` or `/repair` to remove or modify an installation.
+
```

Comment:
Here, I tweaked it to kind of follow the style of the previous list of items

```suggestion
If you've already installed .NET, use the .NET Installer to manage the installation. Instead of `/install`, use one of the following options:

- `/uninstall`\
Remove this version of .NET.

- `/repair`\
Check if the installations key files or components are damaged and restore them.
```